lib/puppet/provider/sensu_entity/sensuctl.rb

```diff
@@ -8,14 +8,7 @@
   def self.instances
     entities = []
 
-    output = sensuctl_list('entity')
-    Puppet.debug("sensu entities: #{output}")
```

Comment:
why has this block been removed?

Reply:
Previously the function `sensuctl_list` would return raw output and each provider had to parse the JSON string.  I moved that common logic into the function and so now every provider just gets back the parsed JSON.  This put the code for parsing JSON and handling edge cases in one place.